src/Chapter2.hs

```diff
@@ -741,7 +753,13 @@ the list with only those lists that contain a passed element.
 
 🕯 HINT: Use the 'elem' function to check whether an element belongs to a list
 -}
-contains = error "contains: Not implemented!"
+contains :: Int -> [[Int]] -> [[Int]]
+contains n l = go n l where
+  go :: Int -> [[Int]] -> [[Int]]
+  go _ [] = []
+  go n (x:xs) = if n `elem` x 
+    then x : go n xs 
+    else go n xs
```

Comment:
Nice `go`! It is actually the standard `filter` function, that you can use instead of the custom `go`, but it is correct!